scrapy/utils/ftp.py

```diff
@@ -13,3 +15,20 @@ def ftp_makedirs_cwd(ftp, path, first_call=True):
         ftp.mkd(path)
         if first_call:
             ftp.cwd(path)
+
+def ftp_store_file( 
+        path, file, host ,port,
+        username, password, use_active_mode=False):
```

Comment:
what do you think about making all argument kwargs-only, to discourage positional argument usage here?

Reply:
I think it would be better yes, I just saw most of the code base use positional arguments so didn't try to be odd 😄  go for it ?

Reply:
go for it :) It is a new function, so no backwards compatibility issues.

Reply:
and thanks for trying to follow the style of the existing code - it makes sense, and makes reviewing easier.

Reply:
Done :)